sprout/sprout/settings.py

```diff
@@ -166,7 +166,7 @@
 
     'free-appliance-shepherd': {
         'task': 'appliances.tasks.free_appliance_shepherd',
-        'schedule': timedelta(minutes=5),
+        'schedule': timedelta(minutes=1),
```

Comment:
@psav And besides of that, unless the shepherd decides to provision/delete anything, it does not touch anything external so it could run even every 10s :)